gapis/memory/memory_pb/memory.proto

```diff
@@ -19,32 +19,22 @@ package memory_pb;
 
 // Observation is the structure for a single memory observation.
 message Observation {
+    // The pool identifier.
+    uint64 pool = 1;
     // Base is the starting address for the observation range.
-    uint64 base = 1;
+    uint64 base = 2;
     // Size is the byte count of the observation range.
-    uint64 size = 2;
-    // Id is the resource id of the bytes observed (used in memory).
-    bytes id = 3;
+    uint64 size = 3;
     // ResIndex is the index of the resource (used on disk).
     sint64 resIndex = 4;
 }
 
-// Pointer is the type representing a memory pointer.
-message Pointer {
-    // The memory address.
-    uint64 Address = 1;
-    // The memory pool.
-    uint32 Pool = 2;
-}
-
 // Slice is the common data between all slice types.
 message Slice {
-    // Original pointer this slice derives from.
-    uint64 Root = 1;
+    // The pool identifier.
+    uint64 Pool = 1;
```

Comment:
Why is pool now a u64?

Reply:
I don't know.  Andrew changed it, I followed.  For symetry with other fields? :-)